csrc/expr_evaluator.cpp

```diff
@@ -55,10 +55,10 @@ void validateValWithConcreteValue(
         actual_dtype);
     if (tv->isCpuScalar()) {
       NVF_CHECK(
-          is_cpu_scalar(t),
+          is_cpu_scalar(t) || is_meta_scalar(t),
```

Comment:
If an intermediate tensorview is CPU scalar, during compilation it is still created as a meta tensor since we do not yet have an actual tensor corresponding to that tensorview.

This occurs in fusions containing SDPA fwd and bwd ops.  The outputs from fwd ops -- `query_seq_len, key_seq_len, philox seed, philox offset` are CPU scalars.

If these are not marked as CPU scalars when writing the fusion, it will lead to an error in binding once the tensors have been computed. If they are marked as CPU scalars, then in the unmodified version, we run into an error in compilation since these intermediate tensors are created as meta tensors.

Reply:
Can you summarize that in a comment at this location?